coalib/processes/Processing.py

```diff
@@ -751,7 +755,12 @@ def execute_section(section,
                              results (bear names are key) as well as the
                              file dictionary.
     """
-    if debug:
+    debug_bears = (False
+                   if 'debug_bears' not in section or (
+                       section['debug_bears'].value == 'False') else (
+                            typed_list(str)(section['debug_bears'])))
```

Comment:
Does it have to be a `typed_list`? That's intended for bear-use only^^

Reply:
Yes `typed_list` is used for the case like `--debug-bears ABear, BBear..` it will split up the the value like `['ABear' 'BBear']`